apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.types.ts

```diff
@@ -12,6 +16,7 @@ export const InstanceQuestionRowSchema = InstanceQuestionSchema.extend({
   assessment_question: AssessmentQuestionSchema,
   user_or_group_name: z.string().nullable(),
   open_issue_count: z.number().nullable(),
+  iq_stable_order: IdSchema,
```

Comment:
Even though this is a "number", since it's derived from an ID it needs to be parsed like one.